alpha/alpha-server/src/main/java/org/apache/servicecomb/pack/alpha/server/api/APIv1Controller.java

```diff
@@ -31,33 +32,36 @@
 
 @RestController
 @RequestMapping("/alpha/api/v1")
-public class APIControllerV1 {
+public class APIv1Controller {
 
   @Autowired
-  AlphaMetrics AlphaMetrics;
-
-  @Autowired(required = false)
-  TransactionRepository transactionRepository;
+  APIv1Impl APIv1Impl;
 
   @GetMapping(value = "/metrics")
   ResponseEntity<AlphaMetrics> metrics() {
-    return ResponseEntity.ok(AlphaMetrics);
+    return ResponseEntity.ok(APIv1Impl.getMetrics());
   }
 
   @GetMapping(value = "/transaction/{globalTxId}")
   ResponseEntity<GlobalTransaction> getTransactionByGlobalTxId(@PathVariable String globalTxId)
       throws Exception {
-    GlobalTransaction globalTransaction = transactionRepository
-        .getGlobalTransactionByGlobalTxId(globalTxId);
-    return ResponseEntity.ok(globalTransaction);
+    return ResponseEntity.ok(APIv1Impl.getTransactionByGlobalTxId(globalTxId));
   }
 
   @GetMapping(value = "/transaction")
   ResponseEntity<PagingGlobalTransactions> getTransactions(@RequestParam(value = "page", required = false, defaultValue = "0") int page,
       @RequestParam(value = "size", required = false, defaultValue = "50") int size)
       throws Exception {
-    PagingGlobalTransactions pagingGlobalTransactions = transactionRepository
-        .getGlobalTransactions(page, size);
-    return ResponseEntity.ok(pagingGlobalTransactions);
+    return ResponseEntity.ok(APIv1Impl.getTransactions(null,page,size));
+  }
+
+  @GetMapping(value = "/transaction/statistics")
+  ResponseEntity<Map<String,Long>> getTransactionStatistics() {
+    return ResponseEntity.ok(APIv1Impl.getTransactionStatistics());
+  }
+
+  @GetMapping(value = "/transaction/slow")
+  ResponseEntity<List<GlobalTransaction>> getSlowTransactions() {
+    return ResponseEntity.ok(APIv1Impl.getSlowTransactions());
```

Comment:
Do you need to add the number to the slow transactions?

Reply:
> Do you need to add the number to the slow transactions?

Yes, I better add a size parameter default value is 10